Sources/PackageCollections/Model/Package.swift

```diff
@@ -67,6 +70,7 @@ extension PackageCollectionsModel {
         init(
             repository: RepositorySpecifier,
             description: String?,
+            keywords: [String]?,
```

Comment:
Is there a semantic distinction to an empty vs a nil collection of keywords, or is it mainly to avoid a coded entry for an empty collection?

Reply:
the latter